Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -2669,7 +2669,11 @@ def generic_api_call_command(client: Client, args: Dict) -> Union[str, CommandRe
     method = str(args.get("method"))
     path = str(args.get("path"))
     headers = json.loads(str(args.get("headers", {})))
-    body: Dict = json.loads(str(args.get("body", {})))
+    json_body = args.get("json_body", "true")
+    if json_body == "true":
+        body: Dict = json.loads(str(args.get("body", {})))
+    else:
+        body = args.get("body", "")
```

Comment:
What other option is there for the body? can the same functionality be achieved by this code? -
```suggestion
   try:
        body: Dict = json.loads(str(args.get("body", {})))
    except ValueError:
        body = args.get("body", "")
```

Reply:
I suppose that's another way to do it. But shouldn't the  caught exception should be a JSONDecodeError?